node/http.ts

```diff
@@ -299,6 +299,12 @@ export class ServerResponse extends NodeWritable {
     }
   }
 
+  /** Returns true if the response body should be null with the given
+   * http status code */
+  static #bodyShouldBeNull(status: number) {
+    return status === 101 || status === 204 || status === 205 || status === 304;
```

Comment:
This condition matches with https://github.com/denoland/deno/blob/8bdf66c59c7424759f441e19047d7ffc0bf13ef3/ext/fetch/23_response.js#L82